src/static/js/components/viewer.js

```diff
@@ -0,0 +1,37 @@
+export default class Viewer {
+  constructor(elem, params = {}) {
+    this.elem = elem;
+    this.params = params;
+
+    this.cssElemType = this.params.cssElemType || '';
+    this.cssElemImage = this.params.cssElemImage || '';
```

Comment:
add console.log(this.cssElemImage) and check what is going on here, what inside this.cssElemImage ?

Reply:
fixed